build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+git clone https://github.com/EFulmer/code-or-die.git
```

Comment:
Are you sure you want to clone from EFulmer?

Reply:
Yeah, good point!